src/Components/Medicine/PrintPreview.tsx

```diff
@@ -42,61 +42,62 @@ export default function PrescriptionsPrintPreview() {
         <h3>{encounter?.facility_name}</h3>
         <img className="h-10 w-auto" src={main_logo.dark} alt="care logo" />
       </div>
-      <div className="mb-6 grid grid-cols-8 gap-y-1.5 border-2 border-secondary-400 p-2">
-        <PatientDetail name="Patient" className="col-span-5">
-          {patient && (
-            <>
-              <span className="uppercase">{patient.name}</span> -{" "}
-              {t(`GENDER__${patient.gender}`)},{" "}
-              {patientAgeInYears(patient).toString()}yrs
-            </>
-          )}
-        </PatientDetail>
-        <PatientDetail name="IP/OP No." className="col-span-3">
-          {encounter?.patient_no}
-        </PatientDetail>
+      <div className="mb-6 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-y-1.5 border-2 border-secondary-400 p-2">
+  <PatientDetail name="Patient" className="col-span-1 md:col-span-2 lg:col-span-2">
+    {patient && (
+      <>
+        <span className="uppercase">{patient.name}</span> -{" "}
+        {t(`GENDER__${patient.gender}`)},{" "}
+        {patientAgeInYears(patient).toString()}yrs
+      </>
+    )}
+  </PatientDetail>
+  <PatientDetail name="IP/OP No." className="col-span-1 md:col-span-1 lg:col-span-1">
+    {encounter?.patient_no}
+  </PatientDetail>
 
-        <PatientDetail
-          name={
-            encounter
-              ? `${t(`encounter_suggestion__${encounter.suggestion}`)} on`
-              : ""
-          }
-          className="col-span-5"
-        >
-          {formatDate(encounter?.encounter_date)}
-        </PatientDetail>
-        <PatientDetail name="Bed" className="col-span-3">
-          {encounter?.current_bed?.bed_object.location_object?.name}
-          {" - "}
-          {encounter?.current_bed?.bed_object.name}
-        </PatientDetail>
+  <PatientDetail
+    name={
+      encounter
+        ? `${t(`encounter_suggestion__${encounter.suggestion}`)} on`
+        : ""
+    }
+    className="col-span-1 md:col-span-2 lg:col-span-2"
+  >
+    {formatDate(encounter?.encounter_date)}
+  </PatientDetail>
+  <PatientDetail name="Bed" className="col-span-1 md:col-span-1 lg:col-span-1">
+    {encounter?.current_bed?.bed_object.location_object?.name}
+    {" - "}
+    {encounter?.current_bed?.bed_object.name}
+  </PatientDetail>
+
+  <PatientDetail name="Allergy to medication" className="col-span-1 md:col-span-2 lg:col-span-4">
+    {patient?.allergies ?? "None"}
+  </PatientDetail>
+</div>
 
-        <PatientDetail name="Allergy to medication" className="col-span-8">
-          {patient?.allergies ?? "None"}
-        </PatientDetail>
-      </div>
 
-      <PrescriptionsTable items={normalPrescriptions} />
+      <PrescriptionsTable items={normalPrescriptions}/>
       <PrescriptionsTable items={prnPrescriptions} prn />
 
-      <div className="pt-12">
-        <p className="font-medium text-secondary-800">
+      <div className="pt-12 px-4">
+        <p className="font-medium text-secondary-800 text-base">
```

Comment:
Let's not make changes to things that were not asked in the issue. 

```suggestion
        <p className="font-medium text-secondary-800">
```